src/main/java/com/attacktimer/AttackTimerBarOverlay.java

```diff
@@ -86,6 +91,20 @@ public Dimension render(Graphics2D graphics)
             ratio = (float)Math.max(1.0f - ratio, 0f);
         }
 
+        if (client.getSpriteOverrides().containsKey(SpriteID.HEALTHBAR_DEFAULT_FRONT_30PX)) {
+            final int barWidth = HD_FRONT_BAR.getWidth();
+            final int barHeight = HD_FRONT_BAR.getHeight();
+            final int barX = canvasPoint.getX() - barWidth / 2;
+            final int barY = canvasPoint.getY();
```

Comment:
also on line  82 replace 

```
final int height = client.getLocalPlayer().getLogicalHeight() + config.heightOffset() - 20;
```

with

```
final int height = client.getLocalPlayer().getLogicalHeight() + config.heightOffset() + 5;
```

Reply:
So it will look better from a side with default settings
Before:
![imagen](https://github.com/user-attachments/assets/9a437f69-3e67-4b09-b8c2-52f1f253fc2c)
![imagen](https://github.com/user-attachments/assets/3784c306-097b-40c2-bcaf-f80910829729)

After:
![imagen](https://github.com/user-attachments/assets/93f3645c-3787-4674-81d1-34937f4ae607)
![imagen](https://github.com/user-attachments/assets/3e24c690-0908-4b8e-b386-b97b9b040a15)